src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -12,7 +12,22 @@ interface Props {
 }
 
 export interface ReleaseStatus {
-  overallStage: string;
+  releaseId?: string;
+  dataStage?: string;
+  contentStage?: string;
+  filesStage?: string;
+  publishingStage?: string;
+  overallStage:
+    | 'Scheduled'
```

Comment:
Nice. This is definitely looking better, but not quite there yet.

Extract another type called `Stage` which includes all of these string literals, then assign it to all the other 'stage' properties.